mergeAxeResults.js

```diff
@@ -120,6 +123,48 @@ const writeResults = async (allissues, storagePath, jsonFilename = 'compiledResu
   }
 };
 
+const writeCsv = async (pageResults, storagePath) => {
+  const csvOutput = createWriteStream(`${storagePath}/reports/report.csv`, { encoding: 'utf8' });
+  const flattenResult = item => {
+    const results = [];
+    const baseObj = { url: item.url };
+    const severities = ['mustFix', 'goodToFix'];
+    for (let severity of severities) {
+      const rules = item[severity].rules;
+      const ruleIds = Object.keys(rules);
+      for (let ruleId of ruleIds) {
+        const rule = rules[ruleId];
+        const { description, helpUrl, conformance, items } = rule;
+        // we filter out the below as it represents the A/AA/AAA level, not the clause itself
+        const clauses = conformance.filter(
+          clause => !['wcag2a', 'wcag2aa', 'wcag2aaa'].includes(clause),
+        );
+        for (let item of items) {
+          const { html, message, page } = item;
+          const context = html ? html : page;
```

Comment:
for `context`, the divs are a little random with the newlines. please help to remove all the new lines